src/main/java/org/openRealmOfStars/utilities/repository/GameRepository.java

```diff
@@ -63,14 +50,12 @@ public void enableJUnit() {
   public void saveGame(final String filename, final StarMap starMap) {
     if (starMap != null) {
       String folderName = "saves";
-      if (runningJUnit) {
-        folderName = "src/test/resources";
-      }
-      File folder = new File(folderName);
+      ClassLoader classLoader = getClass().getClassLoader();
+      File folder = new File(classLoader.getResource(folderName).getFile());
       if (!folder.exists()) {
         folder.mkdirs();
       }
-      File file = new File(folderName + "/" + filename);
+      File file = new File(classLoader.getResource(folderName + "/" + filename).getFile());
```

Comment:
This works for JUnit but not for actual loading saved games. Saved games are outside of JAR so getResource is not going to work.